### DISCUSSION TITLE
Server instance is closed,How to retrieve my posts

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [mastodon/mastodon#23873] [Q&A]
My post at [sinblr.com](https://sinblr.com/@cretaceous), sinblr.com has been closed, and I have not backed up to other instances,
How to retrieve posts

**Comment by nightpool** [comment]
You will have to contact the admin directly, they're responsible for hosting the posts you've made. You can also try signing up for another account on an instance where you might have had followers and see if that instance has retained your posts that you sent them, but most admins clean cached remote media regularly so it may be unlikely.

**Reply by Pantyhose-X** [reply]
I found my post in the new instance, but how do I export the posts and followers of the closed instance?
